src/test/java/teammates/test/cases/ui/StudentFeedbackSubmissionEditSaveActionTest.java

```diff
@@ -779,7 +805,8 @@ public void testExecuteAndPostProcess() throws Exception {
                 Const.ParamsNames.FEEDBACK_RESPONSE_TEXT + "-1-0", fr.getResponseDetails().getAnswerString(),
                 Const.ParamsNames.FEEDBACK_QUESTION_NUMSCALE_MIN + "-1-0", Integer.toString(fqd.getMinScale()),
                 Const.ParamsNames.FEEDBACK_QUESTION_NUMSCALE_MAX + "-1-0", Integer.toString(fqd.getMaxScale()),
-                Const.ParamsNames.FEEDBACK_QUESTION_NUMSCALE_STEP + "-1-0", StringHelper.toDecimalFormatString(fqd.getStep())
+                Const.ParamsNames.FEEDBACK_QUESTION_NUMSCALE_STEP + "-1-0",
+                        StringHelper.toDecimalFormatString(fqd.getStep())
```

Comment:
Not actually, as this is for unregistered student, and for unregistered student we need regKey.